src/main/java/org/broadinstitute/consent/http/service/ElectionService.java

```diff
@@ -384,6 +384,10 @@ public boolean checkDataOwnerToCloseElection(Integer electionId){
         return closeElection;
     }
 
+    public List<Election> findElectionsWithCardHoldingUsersByElectionIds(List<Integer> electionIds) {
+        return electionDAO.findElectionsWithCardHoldingUsersByElectionIds(electionIds);
```

Comment:
Same empty list handling feedback applies here - current usages are safe, this would be a future-proofing kind of thing.